app/src/main/java/org/cru/godtools/util/ViewUtils.kt

```diff
@@ -16,3 +26,26 @@ fun PicassoImageView?.bindLocalImage(attachment: Attachment?) =
     bindLocalImage(attachment?.takeIf { it.isDownloaded }?.localFileName)
 
 fun PicassoImageView?.bindLocalImage(filename: String?) = this?.setPicassoFile(context.getGodToolsFile(filename))
+
+fun TextView.setTextViewWebLinksForAnalytics(
+    changedText: String
+) {
+        val spannable = SpannableString(changedText)
+        val matcher = Patterns.WEB_URL.matcher(spannable)
+        while (matcher.find()) {
+            val url = spannable.toString().substring(matcher.start(), matcher.end())
+            val urlSpan = object : URLSpan(changedText.toString()) {
+                override fun onClick(widget: View) {
+                    val uri = Uri.parse(url)
+                    EventBus.getDefault().post(ExitLinkActionEvent(uri))
+                    context.startActivity(Intent(Intent.ACTION_VIEW, uri))
```

Comment:
use `super.onClick(widget)` here to defer to the framework behavior for launching the external link